backend/src/src-components/src/main/java/org/eclipse/sw360/components/ComponentHandler.java

```diff
@@ -273,10 +273,16 @@ public RequestStatus updateRelease(Release release, User user) throws TException
         assertNotNull(release);
         assertId(release.getId());
         assertUser(user);
-
+        removeSelftLink(release);
         return handler.updateRelease(release, user, ThriftUtils.IMMUTABLE_OF_RELEASE);
     }
 
+    private void removeSelftLink(Release release) {
```

Comment:
Could you please fix the typo real quick?

Reply:
Removed the typo. Sorry for that. 
1. I do not see any ways to add self link during creation as the release is not created yet.
2.As I tested it, if I try to update the release from moderation it redirects me to the same update release flow so the changes are reflecting for me.
3.I will have to check on  fossology releases.
As suggested if I add my code inside updateRelease() then the unit test case will fail due to the immutability check.

Reply:
1. Oh, of course *facepalm* - my bad :D 
2. Sounds good!
3. That would be a question for @mcjaeger what would be correct. If this self link should be excluded from immutability or what's correct here. Then we could just fix the unit test.